.pipelines/templates/testartifacts.yml

```diff
@@ -25,19 +25,18 @@ jobs:
     env:
       ob_restore_phase: true
 
+  - template: /.pipelines/templates/SetVersionVariables.yml@self
+    parameters:
+      ReleaseTagVar: $(ReleaseTagVar)
+
+  - template: /.pipelines/templates/cloneToOfficialPath.yml@self
```

Comment:
We should not be doing this, and definitely not be adding additional places where we do it.

Reply:
Yup, we should prefer using the official ADO task to downloading and running the install script.